networkx/algorithms/community/quality.py

```diff
@@ -246,50 +246,45 @@ def coverage(G, partition):
     total_edges = G.number_of_edges()
     return intra_edges / total_edges
 
-
-def modularity(G, communities, weight='weight'):
+def modularity(G, communities, weight='weight', resolution=1):
     r"""Returns the modularity of the given partition of the graph.
-
     Modularity is defined in [1]_ as
-
     .. math::
-
-        Q = \frac{1}{2m} \sum_{ij} \left( A_{ij} - \frac{k_ik_j}{2m}\right)
+        Q = \frac{1}{2m} \sum_{ij} \left( A_{ij} - \gamma\frac{k_ik_j}{2m}\right)
             \delta(c_i,c_j)
-
     where $m$ is the number of edges, $A$ is the adjacency matrix of
     `G`, $k_i$ is the degree of $i$ and $\delta(c_i, c_j)$
-    is 1 if $i$ and $j$ are in the same community and 0 otherwise.
-
+    is 1 if $i$ and $j$ are in the same community and 0 otherwise,
+    $\gamma$ is the resolution parameter.
     Parameters
     ----------
     G : NetworkX Graph
-
     communities : list
         List of sets of nodes of `G` representing a partition of the
         nodes.
-
+    resolution : float
+        Default 1.
+        If resolution is less than 1, modularity favors largers communities.
+        Greater than 1 favors smaller communities.
     Returns
     -------
     Q : float
         The modularity of the paritition.
-
```

Comment:
These blank lines are needed to allow Sphinx to create the documentation correctly.
Please put back in the blank lines (before each heading).